components/common/NotificationBox/NotificationItem.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <nuxt-link
+    :to="`/${urlPrefix}/gallery/${event.nft.id}`"
+    class="is-flex notify-item">
+    <img
+      v-if="avatar"
+      :src="avatar"
+      :alt="displayName"
+      class="border image is-48x48" />
+    <img
+      v-else
+      src="/placeholder.webp"
+      :alt="displayName"
+      class="border image is-48x48" />
+    <div class="notify-content ml-5 is-flex is-flex-direction-column">
+      <div class="is-flex is-justify-content-space-between">
+        <NeoTooltip
+          :label="displayName"
+          :append-to-body="false"
+          class="nft-name mr-4 has-text-weight-bold">
+          <div class="is-ellipsis">
+            {{ displayName }}
+          </div>
+        </NeoTooltip>
+        <div class="nft-price is-ellipsis">
+          <Money :value="event.meta" />
+        </div>
+      </div>
+      <div class="is-flex is-justify-content-space-between">
+        <div>
+          <div class="height-50px is-flex is-align-items-center">
+            <div
+              class="event-type mr-4 border is-size-7 is-justify-content-center py-1 is-flex is-align-items-center"
+              :class="[
+                getInteractionColor(event.interaction),
+                event.interaction === Interaction.ACCEPTED_OFFER
+                  ? 'px-2'
```

Comment:
it is a bit strange to change padding according to interaction type

Reply:
> mby lower the padding for this specific tag?

https://github.com/kodadot/nft-gallery/pull/5442#issuecomment-1500037007